schemas/rum/resource-schema.json

```diff
@@ -60,6 +60,12 @@
               "minimum": 0,
               "readOnly": true
             },
+            "resolveDuration": {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is it something that we plan to keep?
if not, let's not expose it to customer and put it on `_dd`